src/v3/src/transformer/profile/transformEnrollProfile.test.ts

```diff
@@ -152,23 +136,23 @@ describe('Enroll Profile Transformer Tests', () => {
       .toBe('oie.registration.form.title');
     expect((updatedFormBag.uischema.elements[1] as DescriptionElement).options?.content)
       .toBe('oie.form.field.optional.description');
-    expect(updatedFormBag.uischema.elements[2].type).toBe('PasswordRequirements');
-    expect((updatedFormBag.uischema.elements[2] as PasswordRequirementsElement).options?.id)
+    expect((updatedFormBag.uischema.elements[2] as FieldElement).options?.inputMeta.name)
+      .toBe('userProfile.firstName');
+    expect((updatedFormBag.uischema.elements[3] as FieldElement).options?.inputMeta.name)
+      .toBe('userProfile.lastName');
+    expect((updatedFormBag.uischema.elements[4] as FieldElement).options?.inputMeta.name)
+      .toBe('userProfile.email');
+    expect(updatedFormBag.uischema.elements[5].type).toBe('PasswordRequirements');
+    expect((updatedFormBag.uischema.elements[5] as PasswordRequirementsElement).options?.id)
       .toBe('password-authenticator--list');
-    expect((updatedFormBag.uischema.elements[2] as PasswordRequirementsElement).options?.header)
+    expect((updatedFormBag.uischema.elements[5] as PasswordRequirementsElement).options?.header)
       .toBe('password.complexity.requirements.header');
-    expect((updatedFormBag.uischema.elements[2] as PasswordRequirementsElement).options?.userInfo)
-      .toEqual(mockUserInfo);
-    expect((updatedFormBag.uischema.elements[2] as PasswordRequirementsElement).options?.settings)
+    expect((updatedFormBag.uischema.elements[5] as PasswordRequirementsElement).options?.userInfo)
+      .toEqual({});
+    expect((updatedFormBag.uischema.elements[5] as PasswordRequirementsElement).options?.settings)
       .toEqual({ complexity: { minNumber: 1, minSymbol: 1 } });
-    expect((updatedFormBag.uischema.elements[2] as PasswordRequirementsElement)
+    expect((updatedFormBag.uischema.elements[5] as PasswordRequirementsElement)
```

Comment:
Hard to read. Consider changing to [toEqual](https://jestjs.io/docs/expect#toequalvalue) or [toMatchObject](https://jestjs.io/docs/expect#tomatchobjectobject)?

Reply:
I can update this to make it read easier, however, are you sure you prefer `toMatchObject` over `toEqual`? It seems like toMatchObject would loosen the restrictions that toEqual currently provides, in which case would not fail when significant changes are made. Granted this is a unit test but if someone adds some new properties to this settings object, I feel like it would be better to fail (to notify them that they need to update tests) than to have it pass since they didn't remove any expected keys. 

What do you think?